Svc/PrmDb/test/int/test_cmd_PrmDb.py

```diff
@@ -0,0 +1,91 @@
+""" test_cmd_PrmDb.py:
+
+Test the command dispatcher with basic integration tests.
+"""
+import time
+
+def test_send_PrmDb(fprime_test_api):
+    """Test that commands may be sent
+
+     Tests command send, dispatch, and receipt using send_and_assert command with a pair of CmdDispatcher commands.
+
+     # remove file PrmDB.dat
+
+     recvBuffComp.PARAMETER1_PRM_SET/SAV  
+     recvBuffComp.PARAMETER2_PRM_SET/SAV
+
+     sendBuffComp.PARAMETER3_PRM_SET/SAV  
+     sendBuffComp.PARAMETER4_PRM_SET/SAV
+
+    """
+    
+    
+    # 1st remove file PrmDB.dat (manually)
+    # setup directory to remove later
+    fprime_test_api.send_and_assert_command(fprime_test_api.getCmdDispName('Svc.FileManager') + '.' + 'CreateDirectory', ["/tmp/PrmDb"], max_delay=5)    
+    
+    # send PRM_SAVE_FILE (Wrote 0 records)
+    fprime_test_api.send_and_assert_command(fprime_test_api.getCmdDispName('Svc.PrmDb') + '.' + 'PRM_SAVE_FILE', max_delay=1)
+    # Verify PrmDB.dat is empty    open /tmp/save_file1 (PrmDb.dat file = 0bytes )
+    fprime_test_api.send_and_assert_command(fprime_test_api.getCmdDispName('Svc.FileManager') + '.' + 'ShellCommand', ["ls -lagt", "/tmp/PrmDb/save_file1"], max_delay=5)
+
+
+    ## setup default-value
+    fprime_test_api.send_and_assert_command(fprime_test_api.getCmdDispName('Ref.RecvBuff') + '.' + 'PARAMETER1_PRM_SET', [5], max_delay=5)
+    fprime_test_api.send_and_assert_command(fprime_test_api.getCmdDispName('Ref.RecvBuff') + '.' + 'PARAMETER2_PRM_SET', [6], max_delay=5)
+    fprime_test_api.send_and_assert_command(fprime_test_api.getCmdDispName('Ref.SendBuff') + '.' + 'PARAMETER3_PRM_SET', [7], max_delay=5)
+    fprime_test_api.send_and_assert_command(fprime_test_api.getCmdDispName('Ref.SendBuff') + '.' + 'PARAMETER4_PRM_SET', [8], max_delay=5)
+    
+    # send command PARAMETER1_PRM_SET / SAVE
+    fprime_test_api.send_and_assert_command(fprime_test_api.getCmdDispName('Ref.RecvBuff') + '.' + 'PARAMETER1_PRM_SET', [11], max_delay=5)
+    fprime_test_api.send_and_assert_command(fprime_test_api.getCmdDispName('Ref.RecvBuff') + '.' + 'PARAMETER1_PRM_SAVE', max_delay=1)    
+
+    # Wrote 1 records
+    fprime_test_api.send_and_assert_command(fprime_test_api.getCmdDispName('Svc.PrmDb') + '.' + 'PRM_SAVE_FILE', max_delay=5)
+    # make sure file save
+    time.sleep(2)
+    # Verify PrmDb.dat is increase = 13 bytes     open /tmp/save_file2 (PrmDb.dat file = 13bytes )    
+    fprime_test_api.send_and_assert_command(fprime_test_api.getCmdDispName('Svc.FileManager') + '.' + 'ShellCommand', ["ls -lagt", "/tmp/PrmDb/save_file2"], max_delay=5)        
```

Comment:
Should not use `ShellCommand` this would not work for VxWorks. The script should send a FileDownlink request for the file, once the file is downloaded to the GDS machine, then you can use linux commands in python to get the size.